### PR TITLE
Feature/turkish lbx

### PR DESCRIPTION
**Pull request by abdulkerimgok**
Now all the problems have been solved. Turns out I needed an upgrade. 

The pdf now compiles with bibliography with no problem. 

Only and the biggest problem is the passive strings and according to the developers message there is not any easy solution for that. 

So I am going to propose that we translate all the passive strings as if there were active. 

Let's see what they think.

### CODE REVIEW

**Comment by abdulkerimgok** [comment]
So other than the passive strings, we are almost there. 

Since there isn't easy way to solve the problem with the passive strings, I thought of translating all the passives as if there were active, but what is your opinion? 

The developer also asks for a Turkish bibliography, but do you know what he means with that? I am not sure if that exists. 

Since the semester is over, I want to finish this thing and get it over with.

**Comment by bariserkus** [comment]
> So other than the passive strings, we are almost there.
> 
> Since there isn't easy way to solve the problem with the passive strings, I thought of translating all the passives as if there were active, but what is your opinion?
> 
> The developer also asks for a Turkish bibliography, but do you know what he means with that? I am not sure if that exists.
> 
> Since the semester is over, I want to finish this thing and get it over with.

I am swamped in smtg that needs to get done this weekend. I will get back on this on Monday.

File: tex/latex/biblatex/lbx/03-localization-keys.tex [hunk 1]
```diff
@@ -0,0 +1,719 @@
...
+% This file presents default localization keys for the module specified by
+% the main document language passed to 'babel'.
+%
+\documentclass{article}
+
+% localization modules
+% - ascii
+\usepackage[T1]{fontenc}
+\usepackage[utf8]{inputenc}
+\usepackage[turkish, shorthands=:!]{babel} % The babel module for Turkish defines = as a shorthand that is aimed to leave some space before it in case it's used in text.
```
Comment (bariserkus) [inline tex/latex/biblatex/lbx/03-localization-keys.tex]: It is critical that the Turkish users are informed about the `shorthands=:!` option.

File: tex/latex/biblatex/lbx/turkish.lbx [hunk 2]
```diff
@@ -81,612 +69,463 @@
...
-
-\UndeclareBibliographyExtras{%
-  \restorecommand\mkdaterangecomp
-  \restorecommand\mkdaterangecompextra
-  \restorecommand\mkdaterangeterse
-  \restorecommand\mkdaterangeterseextra
 }
 
 \DeclareBibliographyStrings{%
-  bibliography     = {{Bibliyografya}{Bibliyografya}},
```
Comment (bariserkus) [inline tex/latex/biblatex/lbx/turkish.lbx]: I have capitalized some of the words.

**Pull request merged**